sdk/ml/azure-ai-ml/azure/ai/ml/entities/_util.py

```diff
@@ -191,21 +191,37 @@ def validate_attribute_type(attrs_to_check: dict, attr_type_map: dict):
                 error_type=ValidationErrorType.INVALID_VALUE,
             )
 
+def is_empty_target(obj):
+    """Determines if it's empty target"""
```

Comment:
We won't remove empty string, right?

Reply:
Sure, only remove None, {}, and [].
Actually i'm not sure if [] should be included. Do we support list type as input parameter? if so maybe i should exclude [] here in case user pass [] as parameter value